.github/workflows/tests.yml

```diff
@@ -222,3 +222,71 @@ jobs:
 
       - name: "Tests"
         run: "make tests"
+
+  mutation-testing:
+    name: "Mutation Testing"
+    runs-on: "ubuntu-latest"
+    needs: ["tests", "tests-levels"]
```

Comment:
tests-levels? I don't want Infection to run these levels so it doesn't make sense to wait for them.

Reply:
I thought we could just run PHPStan again before Infection.